src/main/resources/com/google/api/codegen/java/main.snip

```diff
@@ -670,54 +668,61 @@
   {@context.basicTypeNameBoxed(resourcesField.getType())}
 @end
 
-@private initLine(param, docConfig)
-  @let paramType = param.getType()
-    @if paramType.isMessage()
-      @if docConfig.getRequiredParams.isEmpty
-        {@initLineSimple(param, docConfig)}
-      @else
-        {@initLineMessage(param, docConfig)}
-      @end
-    @else
-      {@initLineSimple(param, docConfig)}
+@private initCode(initCodeSpec)
+  @join line : initCodeSpec.getLines()
+    @switch line.getLineType.toString()
+    @case "StructureInitLine"
```

Comment:
The SnippetSet documentation shows them not being indented, so I followed that style:

https://github.com/googleapis/tools-framework/blob/master/src/main/java/com/google/api/tools/framework/snippet/SnippetSet.java#L219